### PR TITLE
Update the CHANGELOG.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

**Comment by ametaireau** [comment]
Whoohoo, 1.0 !

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,18 +4,30 @@ Changelog
...
 
 - Most configuration entries were renamed, see `config/readinglist.ini`
   example to port your configuration.
+- status field was removed, archived and deleted fields were added
+- Remove Python 2.6 support
+
+**New features**
+
+- Add the /fxa-oauth/params endpoint
+- Add the DELETE /articles endpoint
```
Comment (leplatrem) [inline CHANGELOG.rst]: Requires ``cliquet.delete_collection_enabled = true`` in configuration

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,18 +4,30 @@ Changelog
...
 
-0.3 (unreleased)
+1.0 (unreleased)
 ----------------
 
 **Breaking changes**
 
 - Most configuration entries were renamed, see `config/readinglist.ini`
   example to port your configuration.
+- status field was removed, archived and deleted fields were added
```
Comment (ametaireau) [inline CHANGELOG.rst]: nit: Capitalize.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,18 +4,30 @@ Changelog
...
+- Remove Python 2.6 support
+
+**New features**
+
+- Add the /fxa-oauth/params endpoint
```
Comment (ametaireau) [inline CHANGELOG.rst]: nit: sometimes we use dots, sometimes ";", and sometimes nothing, I cannot find any general rule we're applying here. We should define one and follow it :)

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,18 +4,30 @@ Changelog
...
+- Add the DELETE /articles endpoint
+- Add the Response-Behavior header on PATCH /articles
+- Add HTTP calls example in the documentation
```
Comment (ametaireau) [inline CHANGELOG.rst]: `HTTP requests / responses examples`

File: CHANGELOG.rst (hunk 2 above)
Comment (leplatrem) [inline CHANGELOG.rst]: :note:
     requires database flush (no migration supported)

File: docs/versionning.rst [hunk 5]
```diff
@@ -9,7 +9,7 @@ Versionning
 
 The API versioning is based on the application version deployed. It follows `semver <http://semver.org/>`_.
 
-During development the server will be 0.X.X, the server endpoint will be prefixed by ``/v0``.
+During development the server will be 0.X.X, the server endpoint will be prefixed by ``/v1``.
```
Comment (ametaireau) [inline docs/versionning.rst]: This one isn't right. If the server dev version is 0.X.X, then endpoint is /v0.

Comment (Natim) [inline docs/versionning.rst]: In readinglist we are the server version.

File: CHANGELOG.rst (hunk 3 above)
Comment (Natim) [inline CHANGELOG.rst]: In changelog most of the line doesn't have anything at the end, removing all the leading dots.

**Comment by ametaireau** [comment]
r+

**Pull request merged**